weaviate/collection/data.py

```diff
@@ -162,17 +162,19 @@ def _get_by_id(
         )
 
     def _get(
-        self, limit: int, metadata: Optional[GetObjectsMetadata] = None
+        self, limit: Optional[int], metadata: Optional[GetObjectsMetadata]
```

Comment:
why remove the None? Now users have to set them every time? I think both should be None by default

Reply:
Ah sorry, these are the internal methods that are called by the public methods

`_DataCollection.get` and `_DataCollectionModel.get` still have defaults